drivers/sensor/bmi08x/bmi08x_config_file.h

```diff
@@ -0,0 +1,426 @@
+/* Bosch BMI08X inertial measurement unit header
+ *
+ * Copyright (c) 2022 Meta Platforms, Inc. and its affiliates
+ *
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
The source copyright and license should be preserved (Bosch, BSD-3-Clause)

Reply:
fixed.... but it appears to of broken one of the CI checks